tests/baselines/reference/user/uglify-js.log

```diff
@@ -44,7 +49,6 @@ node_modules/uglify-js/lib/compress.js(3235,56): error TS2531: Object is possibl
 node_modules/uglify-js/lib/compress.js(3276,42): error TS2554: Expected 0 arguments, but got 1.
 node_modules/uglify-js/lib/compress.js(3355,33): error TS2532: Object is possibly 'undefined'.
 node_modules/uglify-js/lib/compress.js(3762,38): error TS2554: Expected 0 arguments, but got 1.
-node_modules/uglify-js/lib/compress.js(3788,29): error TS2322: Type '"f"' is not assignable to type 'boolean'.
```

Comment:
errors inside the function expression are squelched because the signature didn't expect arguments. this seems bad.